app/controllers/sign_in/service_account_application_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module SignIn
+  class ServiceAccountApplicationController < ActionController::API
+    include SignIn::Authentication
+    include SignIn::ServiceAccountAuthentication
+    include Pundit::Authorization
+    include ExceptionHandling
+    include Headers
+    include SentryLogging
+    include SentryControllerLogging
+
+    before_action :authenticate_service_account
+    skip_before_action :authenticate
+
+    private
+
+    attr_reader :current_user
```

Comment:
I don't think we need a `current_user` for service account stuff so you can probably remove this

Reply:
`current_user` gets used by CllientConfig and when I remove this attr_reader it throws an error in Sentry, not sure what's originating it.

Reply:
Can you link the Sentry error? I'll see what's causing it. Also, where does it get used by Client Config? I think that might be unexpected

Reply:
```ruby
[44419:puma srv tp 003 sentry_logging.rb:57] Rails -- undefined local variable or method `current_user' for #<V0::AccountControlsController:0x00000000073500>.
[44419:puma srv tp 003 sentry_logging.rb:57] Rails -- /home/john/Projects/va/vets-api/app/controllers/concerns/sentry_controller_logging.rb:15:in `set_tags_and_extra_context'
```

The line in question has a safe operator and I didn't raise and error when I stepped through it with the debugger, so I can't figure out why it's triggering for regular credential_index/lock requests.

Reply:
I see, `current_user` is throwing errors because the variable itself was never defined (if you remove the `attr_reader` variable). I think we can keep it for now but we'll want to have another ticket that properly deals with this in the included `exception_handling`, `sentrycontrollerlogging` files (and any other that may also have this issue)

Reply:
Okay, I'm tracking this and Dick's suggestion for spec improvements and added them to the [followup ticket](https://app.zenhub.com/workspaces/identity-5f5bab705a94c9001ba33734/issues/gh/department-of-veterans-affairs/va.gov-team/67340).